sdk/bpf/c/inc/solana_sdk.h

```diff
@@ -51,6 +51,11 @@ static_assert(sizeof(uint64_t) == 8);
  */
 #define NULL 0
 
+/**
+ * SUCCESS return value
+ */
+#define SUCCESS 0
```

Comment:
Think we should namespace this?

Reply:
Sure, any preference?
* `SOLANA_SUCCESS`
* `PROGRAM_SUCCESS`